usagov/vote-gov-drupal
# Feature/vote 902/header migration

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[VOTE 902](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-902&assignee=609161faf6c0960069bf61b3)
[VOTE 903](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-903&search=footer)

## Description (optional)
Migration of header translations

### Testing steps
1. Pull branch 
2. Import database that includes migrated header and footer translations
3. Run lando retune
4. Confirm that all relevant translations are available for header and footer components

Header: 
1. Government banner 
2. Language selector 
3. Sitewide alert 
4. Search

Footer:
1. Email signup 
2. Footer contact 
3. Footer identifier: agency, menu, link
